orbital-camera.ts

```diff
@@ -75,12 +75,11 @@ export class OrbitalCamera extends Component {
     onActivate(): void {
         const canvas = this.engine.canvas;
 
-        canvas.addEventListener('mousemove', this._onMouseMove);
+        window.addEventListener('mousemove', this._onMouseMove);
```

Comment:
`mousemove` should also go in `mousedown` except if you use it for something else?